### ISSUE TITLE
Season modularity 

### ISSUE THREAD

**Issue by RyGuy101** [JuicyData/TheOrangeAlliance2016-2017#9]
'Python and PHP only compatible with data from the Velocity Vortex game.'
We need the code and the database to work with next year's game, so we should add modularity for the game objectives.
-make a collection with documents detailing game objectives and point values
-base all relevant data-manipulation code on the document structure of the game objectives
@TheOrangeAllianceNet/backend @TheOrangeAllianceNet/frontend

**Comment by MichaelLeonffu** [comment]
I agree with you @RyGuy101 though.......... we have to teach new kids how to do stuff so maybe after that we can make modularity. 🙃
